index.js

```diff
@@ -14,6 +19,11 @@ exports.shapify = (mapper, obj) => {
   return res
 }
 
+const arrayMapper = (keys, obj) => {
+  const mapper = keys.reduce((acc, key) => ({ ...acc, [key]: key }), {})
```

Comment:
I'm adapting the code to be more performant though, I don't want to use reduce nor creating a copy at each iteration